src/components/ADempiere/Form/BarcodeReader/index.vue

```diff
@@ -56,21 +56,36 @@
               <div class="product-price-base">
                 Precio Base
                 <span class="amount">
-                  {{ formatPrice(productPrice.priceBase, productPrice.currency.iSOCode) }}
+                  {{
+                    formatPrice({
+                      value: productPrice.priceBase,
+                      currencyCode
+                    })
+                  }}
                 </span>
               </div>
               <br><br><br>
 
               <div class="product-tax">
                 {{ productPrice.taxName }}
                 <span class="amount">
-                  {{ formatPrice(productPrice.taxAmt, productPrice.currency.iSOCode) }}
+                  {{
+                    formatPrice({
+                      value: productPrice.taxAmt,
+                      currencyCode
+                    })
+                  }}
                 </span>
               </div>
               <br><br><br>
 
               <div class="product-price amount">
-                {{ formatPrice(productPrice.grandTotal, productPrice.currency.iSOCode) }}
+                {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes